src/main/steps/applicant1/finalising-your-application/post.ts

```diff
@@ -20,6 +26,10 @@ export default class FinalisingYourApplicationPostController extends PostControl
   }
 
   protected getEventName(req: AppRequest<AnyObject>): string {
-    return req.session.isApplicant2 ? APPLICANT2_FINAL_ORDER_REQUESTED : FINAL_ORDER_REQUESTED;
+    return needsToExplainDelay(req.session.userCase)
+      ? CITIZEN_APPLICANT2_UPDATE
```

Comment:
is this controller used for both app1 and app2

Reply:
App2 functionality hasn't been implemented yet but it will be.